core/coreapi/unixfs_test.go

```diff
@@ -707,23 +707,6 @@ func TestGetNonUnixfs(t *testing.T) {
 	}
 }
 
-func TestCatOffline(t *testing.T) {
```

Comment:
This brings up an interesting issue: `ipfs cat` currently works offline (even with IPNS) but `ipfs get` doesn't. After this change, they both work offline.

Should we *not* make this change?

Reply:
I'd say that they both should work offline. The only downside is that it might confuse some users because `/ipns` may resolve to old values in datastore without even trying to reach the network (on the other hand we do that with cache too, so..). Maybe we should log a warning from `offroute.GetValue`?